releases/CR4.html

```diff
@@ -1073,14 +1073,15 @@ <h2 id="x2.-conformance"><span class="secno">2. </span>Conformance</h2>
-        Upload</a></dfn> state, <dfn data-dfn-type="dfn" id="dfn-enumerated-attribute"><a href="https://html.spec.whatwg.org/multipage/#enumerated-attribute">enumerated attribute</a></dfn>,
-            <dfn data-dfn-type="dfn" id="dfn-missing-value-default"><a href="https://html.spec.whatwg.org/multipage/#missing-value-default">missing value
-        default</a></dfn>, <dfn data-dfn-type="dfn" id="dfn-invalid-value-default"><a href="https://html.spec.whatwg.org/multipage/#invalid-value-default">invalid
-        value default</a></dfn>, and <dfn data-dfn-type="dfn" id="dfn-reflect"><a href="https://html.spec.whatwg.org/multipage/#reflect">reflect</a></dfn> are defined in [<cite><a class="bibref" href="#bib-HTML51">HTML51</a></cite>].
+            The <dfn data-dfn-type="dfn" id="dfn-input"><a href="https://www.w3.org/TR/html51/sec-forms.html#the-input-element"><code>input</code></a></dfn> element, its <dfn data-dfn-type="dfn" id="dfn-type"><a href="https://www.w3.org/TR/html51/sec-forms.html#element-attrdef-input-type"><code>type</code></a></dfn>            attribute,
+            <dfn data-dfn-for="" data-dfn-type="dfn" id="dom-htmlinputelement" data-idl="" data-title="HTMLInputElement"><a href="https://www.w3.org/TR/html51/sec-forms.html#htmlinputelement-htmlinputelement"><code>HTMLInputElement</code></a></dfn> interface,
+            <dfn data-dfn-type="dfn" id="dfn-accept"><a href="https://www.w3.org/TR/html51/sec-forms.html#element-attrdef-input-accept"><code>accept</code></a></dfn> attribute, <dfn data-dfn-type="dfn" id="dfn-file-upload"><a href="https://www.w3.org/TR/html51/sec-forms.html#element-statedef-input-file-upload">File
+        Upload</a></dfn> state, <dfn data-dfn-type="dfn" id="dfn-enumerated-attribute"><a href="https://www.w3.org/TR/html51/infrastructure.html#enumerated-attributes">enumerated attribute</a></dfn>,
+            <dfn data-dfn-type="dfn" id="dfn-missing-value-default"><a href="https://www.w3.org/TR/html51/infrastructure.html#missing-value-default">missing value
+        default</a></dfn>, <dfn data-dfn-type="dfn" id="dfn-invalid-value-default"><a href="https://www.w3.org/TR/html51/infrastructure.html#invalid-value-default">invalid
+        value default</a></dfn>, and <dfn data-dfn-type="dfn" id="dfn-reflect"><a href="https://www.w3.org/TR/html51/infrastructure.html#reflection">reflect</a></dfn> are defined in [<cite><a class="bibref" href="#bib-HTML51">HTML51</a></cite>].
         </p>
+        <p>The <dfn cite="!custom-elements#cereactions" data-dfn-type="dfn" id="dfn-x[cereactions]"><code>[CEReactions]</code></dfn> WebIDL extended attribute is defined in [<cite><a class="bibref" href="#bib-custom-elements">custom-elements</a></cite>].</p>
```

Comment:
@dontcallmedom, this should be `data-cite`, otherwise does not pass validator https://validator.w3.org/nu/?doc=https://rawgit.com/w3c/html-media-capture/3fbc8ca4a54d05808efaeede78ba9e8a6ce74bef/releases/CR4.html

It seems your respec patch assumed `cite`, does not work with `data-cite`.

I will patch CR4.html manually and you can fix the bug in respec accordingly.